README.md

```diff
@@ -87,6 +87,8 @@ Options:
         --[no-]native-full-aot       Enable/disable NATIVE_FULL_AOT / Ahead of Time compilation (default: disabled)
         --[no-]relink-eln-files      Enable/disable re-linking shared libraries in bundled *.eln files (default: enabled)
         --[no-]rsvg                  Enable/disable SVG image support via librsvg (default: enabled)
+        --[no-]poll                  Enable/disable experimental use of poll() instead of select() to support > 1024 file descriptors (default: disabled)
+        --fd-setsize 2048            Set an arbitrary file descriptor size. To be used in tandem with the `--poll` option (default: disabled)
```

Comment:
Please correct me if I'm wrong, but my understanding is that the `FD_SETSIZE` approach is the way to increase max open files without the poll patch. While the poll patch makes Emacs adhere to system level limits.

Hence I imagine you'd use one or the other, as I'd imagine the poll patch makes the FD_SETSIZE approach do nothing.

Reply:
No, you're entirely correct! Originally I had a misunderstanding about this, and I didn't make a comparable change in this description, as in the other place it is. I've pushed up a fixup commit (that I'll squash).